coders/heic.c

```diff
@@ -242,9 +242,9 @@ static MagickBooleanType ReadHEICExifProfile(Image *image,
       length=GetStringInfoLength(exif_profile);
       datum=GetStringInfoDatum(exif_profile);
       if ((length > 2) && 
-          ((memcmp(datum,"\0xFF\0xD8",2) == 0) ||
-           (memcmp(datum,"\0xFF\0xE1",2) == 0)) &&
-           memcmp(datum+length-2,"\0xFF\0xD9",2) == 0)
+          ((memcmp(datum,"\xFF\xD8",2) == 0) ||
```

Comment:
Could you change this to lowercase since this is the style that we are using in the rest of the project?

Reply:
Done